public/assets/jquery-ui/autocomplete.js

```diff
@@ -28,7 +28,7 @@ function(e){"function"==typeof define&&define.amd?define(["jquery"],e):e(jQuery)
,my:o.my,at:o.at,within:x,elem:c})}),o.using&&(u=function(t){var e=y.left-_.left,i=e+g-h,n=y.top-_.top,r=n+m-p,l={target:{element:w,left:y.left,top:y.top,width:g,height:m},element:{element:c,left:_.left,top:_.top,width:h,height:p},horizontal:0>i?"left":e>0?"right":"center",vertical:0>r?"top":n>0?"bottom":"middle"};h>g&&a(e+i)<g&&(l.horizontal="center"),p>m&&a(n+r)<m&&(l.vertical="middle"),s(a(e),a(i))>s(a(n),a(r))?l.important="horizontal":l.important="vertical",o.using.call(this,t,l)}),c.offset(t.extend(_,{using:u}))})},t.ui.position={fit:{left:function(t,e){var i,n=e.within,o=n.isWindow?n.scrollLeft:n.offset.left,r=n.width,a=t.left-e.collisionPosition.marginLeft,l=o-a,u=a+e.collisionWidth-r-o;e.collisionWidth>r?l>0&&0>=u?(i=t.left+l+e.collisionWidth-r-o,t.left+=l-i):u>0&&0>=l?t.left=o:l>u?t.left=o+r-e.collisionWidth:t.left=o:l>0?t.left+=l:u>0?t.left-=u:t.left=s(t.left-a,t.left)},top:function(t,e){var i,n=e.within,o=n.isWindow?n.scrollTop:n.offset.top,r=e.within.height,a=t.top-e.collisionPosition.marginTop,l=o-a,u=a+e.collisionHeight-r-o;e.collisionHeight>r?l>0&&0>=u?(i=t.top+l+e.collisionHeight-r-o,t.top+=l-i):u>0&&0>=l?t.top=o:l>u?t.top=o+r-e.collisionHeight:t.top=o:l>0?t.top+=l:u>0?t.top-=u:t.top=s(t.top-a,t.top)}},flip:{left:function(t,e){var i,n,o=e.within,r=o.offset.left+o.scrollLeft,s=o.width,l=o.isWindow?o.scrollLeft:o.offset.left,u=t.left-e.collisionPosition.marginLeft,c=u-l,h=u+e.collisionWidth-s-l,p="left"===e.my[0]?-e.elemWidth:"right"===e.my[0]?e.elemWidth:0,d="left"===e.at[0]?e.targetWidth:"right"===e.at[0]?-e.targetWidth:0,f=-2*e.offset[0];0>c?(i=t.left+p+d+f+e.collisionWidth-s-r,(0>i||i<a(c))&&(t.left+=p+d+f)):h>0&&(n=t.left-e.collisionPosition.marginLeft+p+d+f-l,(n>0||a(n)<h)&&(t.left+=p+d+f))},top:function(t,e){var i,n,o=e.within,r=o.offset.top+o.scrollTop,s=o.height,l=o.isWindow?o.scrollTop:o.offset.top,u=t.top-e.collisionPosition.marginTop,c=u-l,h=u+e.collisionHeight-s-l,p="top"===e.my[1],d=p?-e.elemHeight:"bottom"===e.my[1]?e.elemHeight:0,f="top"===e.at[1]?e.targetHeight:"bottom"===e.at[1]?-e.targetHeight:0,g=-2*e.offset[1];0>c?(n=t.top+d+f+g+e.collisionHeight-s-r,(0>n||n<a(c))&&(t.top+=d+f+g)):h>0&&(i=t.top-e.collisionPosition.marginTop+d+f+g-l,(i>0||a(i)<h)&&(t.top+=d+f+g))}},flipfit:{left:function(){t.ui.position.flip.left.apply(this,arguments),t.ui.position.fit.left.apply(this,arguments)},top:function(){t.ui.position.flip.top.apply(this,arguments),t.ui.position.fit.top.apply(this,arguments)}}},function(){var e,i,n,o,s,a=document.getElementsByTagName("body")[0],l=document.createElement("div");e=document.createElement(a?"div":"body"),n={visibility:"hidden",width:0,height:0,border:0,margin:0,background:"none"},a&&t.extend(n,{position:"absolute",left:"-1000px",top:"-1000px"});for(s in n)e.style[s]=n[s];e.appendChild(l),i=a||document.documentElement,i.insertBefore(e,i.firstChild),l.style.cssText="position: absolute; left: 10.7432222px;",o=t(l).offset().left,r=o>10&&11>o,e.innerHTML="",i.removeChild(e)}()}(),t.ui.position}),/*!
```

Comment:
__[jshint]__
* Expected an assignment or function call and instead saw an expression.

<!-- Comment by [SideCI](https://sideci.com) -->